.travis.yml

```diff
@@ -1,13 +1,14 @@
 language: scala
 sudo: false
 scala:
-  - "2.11.11"
+  - "2.11.12"
+  - "2.12.4"
 jdk:
   - oraclejdk8
 script:
   # make 'git branch' work again
   - git branch -f "$TRAVIS_BRANCH" && git checkout "$TRAVIS_BRANCH"
-  - sbt -jvm-opts .travis-jvmopts +test
```

Comment:
The `+` in `+test` makes sure the tests are ran across multiple Scala versions (see the [sbt docs](http://www.scala-sbt.org/1.x/docs/Cross-Build.html#Cross-Building+a+Project) and [our config](https://github.com/akka/reactive-kafka/blob/master/build.sbt#L43)) - so as far as I can see we're already testing against both Scala versions, and this change does not appear to be needed?

Reply:
I just pushed an update.   I  changed "+test" to "test"